Source/BSN.Commons.Orm.Redis/RepositoryBase.cs

```diff
@@ -98,7 +131,8 @@ public T GetById<KeyType>(KeyType id)
             if (id is string str_id)
             {
                 T? entity = dbCollection.FindById(str_id);
-                if (entity == null)
+                T DefaultEntity = JsonSerializer.Deserialize<T>("{}");
```

Comment:
Why this logic is needed?

Reply:
As FindById(str_id) does not return null in case of inexistence of object and it returns a default object, I created a new default object ( JsonSerializer.Deserialize<T>("{}");) and checked if two objects are equal.